tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -4914,6 +4920,9 @@ def get_float32(a):
     assert f_bl_true_output == test_bool_true_output
     assert f_bl_false_output == test_bool_false_output
 
+    assert (type(f_bl_true_output) == type(test_bool_true_output.item()))
+    assert (type(f_bl_false_output) == type(test_bool_false_output.item()))
```

Comment:
Please use the new `matching_types` function whenever you need `.item` otherwise you will break python tests